patchwork/templates/patchwork/submission.html

```diff
@@ -266,6 +266,24 @@ <h2>Message</h2>
   </pre>
 </div>
 
+{% for note in notes %}
+{% if forloop.first %}
+<h2>Notes</h2>
+{% endif %}
+<a name="{{ item.id }}"></a>
+<div class="submission-message">
```

Comment:
Can we use a different class and set the background to a different colour so it stands out? These are presumably rather important.

Reply:
![image](https://github.com/user-attachments/assets/2a394c3e-9235-4de2-affe-98217de11f27)

I used the same style that we have for commit messages since they have a similar level of importance. Do you have any suggestions on how to style it differently ?